plugins/node/opentelemetry-instrumentation-koa/examples/src/server.ts

```diff
@@ -2,11 +2,13 @@
 
 const api = require('@opentelemetry/api');
 
-require('./tracer')('example-koa-server');
+import { setupTracing } from './tracer'
+setupTracing('example-koa-server');
 
 // Adding Koa router (if desired)
 const router = require('@koa/router')();
```

Comment:
nit: also here, might be nice to use ES modules imports consistently and not mix require and import :)